packages/core/strapi/src/cli/commands/cloud/deploy-project.ts

```diff
@@ -0,0 +1,147 @@
+import { createCommand } from 'commander';
+import fs from 'fs';
+import path from 'path';
+import type { CLIContext, StrapiCommand } from '../../types';
+import { runAction } from '../../utils/helpers';
+import { apiConfig } from './config/api';
+import { getStoragePath } from './config/local';
+import { action as createProjectAction } from './create-project';
+import { cloudApiFactory, ProjectInfos } from './services/cli-api';
+import * as localSave from './services/strapi-info-save';
+import { compressFilesToTar } from './utils/compress-files';
+import { notificationServiceFactory } from './utils/notification-service';
+import { tokenServiceFactory } from './utils/token';
+
+const FILE_SIZE_LIMIT = 100 * 1024 * 1024; // 100MB
+
+function getPackageJson({ logger }: CLIContext, folderPath: string): Record<string, any> | null {
```

Comment:
We have a function for this: https://github.com/strapi/strapi/blob/b003dee48ebdf2a6462c1160e1d326b60f36c026/packages/core/strapi/src/cli/utils/pkg.ts#L58

Could we use this here?